ArcGISRuntimeSDKQt_CppSamples/Layers/IdentifyKmlFeatures/README.md

```diff
@@ -0,0 +1,47 @@
+# Identify KML features
+
+Show a callout with formatted content for a KML feature.
+
+![](screenshot.png)
+
+## Use case
+
+A user may wish to select a KML feature to view relevant information about it.
+
+## How to use the sample
+
+Click a feature to identify it. Feature information will be displayed in a callout.
+
+Note: the KML layer used in this sample contains a screen overlay. The screen overlay contains a legend and the logos for NOAA and the NWS. You can't identify the screen overlay.
```

Comment:
should probably expand what the NOAA and NWS acronyms are (I have no idea)

Reply:
@ldanzinger thoughts on adding more explanation to the README?

Reply:
@sbiswas96 in cases like this, I'd suggest updating it here first, then updating the common design. We will have a process to bubble out these changes elsewhere later